cogs/socialcredit.py

```diff
@@ -0,0 +1,138 @@
+from enum import Enum
+from typing import cast
+
+from discord import Color, Embed
+from discord.ext import commands
+from discord.ext.commands.context import Context
+from discord.raw_models import RawReactionActionEvent
+
+from configs import configs
+from db.socialcredit import UserCredit
+from utils.context import get_member
+
+from .base import CommandBase
+
+
+class ScoreDirection(Enum):
+    POSITIVE = 1
+    NEGATIVE = 0
+
+
+class SocialCredit(CommandBase):
+    @commands.Cog.listener()
+    async def on_raw_reaction_add(self, reaction: RawReactionActionEvent):
```

Comment:
`on_reaction_add` wasn't working. something to do with caching. might work depending on intents or something